packages/testhelper/testhelper.go

```diff
@@ -1,27 +1,29 @@
 package testhelper
 
 import (
-	"fmt"
-	"os"
+	"flag"
```

Comment:
using `spf13/pflag` doesn't work well in tests
this has to do with go using `flag` internally